content/engine/release-notes/27.1.md

```diff
@@ -21,6 +21,22 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](../api/version-history.md).
 
+## 27.1.1
+
+{{< release-date date="2024-07-23" >}}
+
+### Security
+
+This release contains a fix for [CVE-2024-41110](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-41110) / [GHSA-v23v-6jw2-98fq](https://github.com/moby/moby/security/advisories/GHSA-v23v-6jw2-98fq)
+that impacted setups using [authorization plugins (AuthZ)](https://docs.docker.com/engine/extend/plugins_authorization/)
+for access control. No other changes are included in this release, and this
+release is otherwise identical for users not using AuthZ plugins.
+
+### Packaging updates
+
+- Update Compose to [v2.29.0](https://github.com/docker/compose/releases/tag/v2.29.0). [moby/docker-ce-packaging#1038](https://github.com/docker/docker-ce-packaging/pull/1038)
```

Comment:
🤦 copied this from the previous release, and forgot to update the version and URL; fixing in https://github.com/docker/docs/pull/20465